### DISCUSSION TITLE
科技爱好者周刊（第 316 期）：你一生的故事 | ruanyf-weekly

### DISCUSSION THREAD

**Discussion by giscus** [plantree/press-comment#43] [General]
# 科技爱好者周刊（第 316 期）：你一生的故事 | ruanyf-weekly

阮一峰的技术周刊

https://ruanyf-weekly.plantree.me/weekly/issue-316

**Comment by Allen277** [comment]
以前是不是没有评论功能？

**Reply by plantree** [reply]
一直都有呀~
